eahub/config/settings.py

```diff
@@ -25,6 +25,7 @@
 DEFAULT_FROM_EMAIL = "EA Hub <noreply@eahub.org>"
 EMAIL_SUBJECT_PREFIX = "[EA Hub] "
 MANAGERS = ADMINS
+GROUP_ADMIN = 'lean@eahub.org'
```

Comment:
This setting should be in the Local Groups section, and it should be an environment variable with a corresponding `docker-compose.yml` entry (which should be something like `groupadmin@example.com`).

Reply:
done